x-pack/plugins/infra/public/lib/lib.d.ts

```diff
@@ -122,6 +130,12 @@ export interface InfraWaffleMapGroupOfNodes extends InfraWaffleMapGroupBase {
   nodes: InfraWaffleMapNode[];
 }
 
-export interface InfraWaffleOptions {
-  [name: string]: any;
+export interface InfraOptions {
```

Comment:
This currently looks like `InfraWaffleOptions` would actually reflect the purpose well :thinking: What was your thinking behind the name change?